Comp2023-Imported/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -186,12 +186,21 @@ public void updateSwerveOdometry( )
 
       //Adding a position specified by the limelight to the estimator at the time that the pose was generated 
       if (botLLPose != null && DriverStation.isTeleopEnabled( ))
+      {
         m_poseEstimator.addVisionMeasurement(botLLPose, Timer.getFPGATimestamp( ) - (latency / 1000));
+        m_poseEstimator.update(m_heading, getPositions( ));
```

Comment:
I think this .update call should already be called within the swerve library provided by CTRE (or else it would not have worked at all even when the LL wasn't there). So this might be applying the update a second time.